packages/debugger-extension/schema/main.json

```diff
@@ -3,6 +3,17 @@
   "description": "Debugger settings",
   "jupyter.lab.setting-icon": "ui-components:bug",
   "jupyter.lab.setting-icon-label": "Debugger",
+  "jupyter.lab.menus": {
```

Comment:
Looks like the visual regression check failure might be relevant, if this PR modifies the menu?

https://github.com/jupyterlab/jupyterlab/pull/10462/checks?check_run_id=2942552592

Reply:
@JohanMabille maybe a new rebase onto the latest could be useful, just to double check?

Reply:
It indeed requires to update the reference screenshot.

Think a bit more about it, I think it will make more sense to add the menu entry in the Kernel menu where are all `Restart and...` commands. what do you think @jtpio ?

Reply:
Sounds like it would make sense to have it there yes :+1:

Reply:
Done